lib/src/scss-shared/_nx-variables.scss

```diff
@@ -13,6 +13,7 @@ $nx-main-header-height: 72px;
 $nx-paragraph-width-maximum: 562px;
 $nx-form-label-width-maximum: 562px;
 
+$nx-scrollbar-width: 15px;
```

Comment:
We already have this as a CSS variable.  Also what does it have to do with this PR?

Reply:
> We already have this as a CSS variable. Also what does it have to do with this PR?

Seems to be an artifact from a merge conflict that I resolved incorrectly: https://github.com/sonatype/sonatype-react-shared-components/pull/343/commits/031e55381c1f87e5bc35c0dff85c0027cfcb6e10